tests/regression/77-lin2vareq/31-function-call.c

```diff
@@ -0,0 +1,20 @@
+// SKIP PARAM: --set ana.activated[+] lin2vareq --set sem.int.signed_overflow assume_none
+
+int myfunction(int x, int y){
+   if (x == 0) {
+    __goblint_check(x == 0); // SUCCESS
+  } else if (y - x == 3) {
+    __goblint_check(y == x + 0); // FAILURE
+    __goblint_check(y - x == 3); // SUCCESS
+  }
+
+  return 5;
```

Comment:
All the relational information is established in the function where the assertions are made. Can you add a test case where the relational information is established in the caller, and the assertions are made in the callee?
Also testing relations w.r.t. to the return value of a function would be interesting.